src/goto-instrument/contracts/instrument_spec_assigns.cpp

```diff
@@ -541,7 +544,7 @@ exprt instrument_spec_assignst::target_validity_expr(
   // the target's `start_address` pointer satisfies w_ok with the expected size
   // (or is NULL if we allow it explicitly).
   // This assertion will be falsified whenever `start_address` is invalid or
-  // not of the right size (or is NULL if we dot not allow it expliclitly).
+  // not of the right size (or is NULL if we dot not allow it explicitly).
```

Comment:
Since you're fixing typos - can you also change "dot" to "do" in here?

Reply:
Yes sure, will do.